assets/js/admin/wc-product-export.js

```diff
@@ -0,0 +1,86 @@
+/*global ajaxurl, wc_product_export_params */
+;(function ( $, window ) {
+	/**
+	 * productExportForm handles the export process.
+	 */
+	var productExportForm = function( $form ) {
+		this.$form = $form;
+		this.xhr   = false;
+
+		// Initial state.
+		this.$form.find('.woocommerce-exporter-progress').val( 0 );
+
+		// Methods.
+		this.processStep = this.processStep.bind( this );
+
+		// Events.
+		$form.on( 'submit', { productExportForm: this }, this.onSubmit );
+	};
+
+	/**
+	 * Handle export form submission.
+	 */
+	productExportForm.prototype.onSubmit = function( event ) {
+		event.preventDefault();
+		event.data.productExportForm.$form.addClass( 'woocommerce-exporter__exporting' );
+		event.data.productExportForm.$form.find('.woocommerce-exporter-progress').val( 0 );
+		event.data.productExportForm.$form.find('.woocommerce-exporter-button').prop( 'disabled', true );
+		event.data.productExportForm.processStep( 1, $( this ).serialize(), '' );
+	};
+
+	/**
+	 * Process the current export step.
+	 */
+	productExportForm.prototype.processStep = function( step, data, columns ) {
+		var $this = this,
+			selected_columns = $( '.woocommerce-exporter-columns' ).val(),
+			export_meta      = $( '#woocommerce-exporter-meta:checked' ).length ? 1 : 0,
```

Comment:
If is just to make bool, then could just use `$( '#woocommerce-exporter-meta:checked' ).length > 0`